### PR TITLE
PostNorm: Parse srcsets and make them safe.

### LINKED ISSUE #2163: Reader: Images that use srcsets from external domains break
See https://wordpress.com/read/post/feed/41826258/898319035

screenshot

The problem is that these images are coming from WP 4.4, which is now setting `srcset` natively. We're not doing anything to protect or strip srcset, so images are coming in without going through the photon checks.

### PR DESCRIPTION
**Pull request by blowery**
Fixes #2163 

To test, pull up http://calypso.localhost:3000/read/post/feed/41826258/898319035, inspect the images in the body of the post and notice that both the `src` and `srcset` elements have been rewritten to use photon.

### CODE REVIEW

File: client/lib/post-normalizer/index.js [hunk 1]
```diff
@@ -509,6 +511,7 @@ normalizePost.content = {
 					// fun fact: removing the node from the DOM will not prevent it from loading. You actually have to
 					// change out the src to change what loads. The following is a 1x1 transparent gif as a data URL
 					image.setAttribute( 'src', 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7' );
+					image.removeAttribute( 'srcset' );
```
Comment (aduth) [inline client/lib/post-normalizer/index.js]: Does `srcset` really load if not inserted into the DOM? Better safe than sorry, but now you've got me curious :smile:

**Comment by aduth** [comment]
Looks good, works well :+1:

File: client/lib/post-normalizer/index.js (hunk 1 above)
Comment (blowery) [inline client/lib/post-normalizer/index.js]: you know... i wonder if what i was seeing was a srcset being present along with a src and that tricked me into thinking it was the src loading the image..

assume you're referring to the comment above

**Comment by rralian** [comment]
late to the party, but also looks good, thanks for tackling @blowery

**Pull request merged**